packages/grafana-ui/src/components/VizTooltip/VizTooltipRow.tsx

```diff
@@ -31,6 +38,25 @@ export const VizTooltipRow = ({
 
   const [showLabelTooltip, setShowLabelTooltip] = useState(false);
   const [showValueTooltip, setShowValueTooltip] = useState(false);
+  const [copiedText, setCopiedText] = useState<Record<string, string> | null>(null);
+
+  const labelRef = useRef<null | HTMLDivElement>(null);
+  const valueRef = useRef<null | HTMLDivElement>(null);
+
+  const copyToClipboard = async (text: string, type: LabelValueTypes) => {
+    if (!navigator?.clipboard) {
+      return false;
```

Comment:
Very much think that this silent failure here is not a good idea. If a user moves between browsers at all and one doesn't support it this will be extremely confusing. Maybe even a tooltip that says "click to copy" or something like that.

Would also say that in this case it would make sense to not have a pointer cursor to make clear it's a non-interactive element in cases where the clipboard isn't accessible.

Reply:
Hey Kyle! I added a fallback method in case Clipboard API is not supported, supposedly should add better support. I couldn't figure out how to disable copying in browser in order to test this out, please let me know if you have any idea.

As for the tooltip, I agree it would be nice to have some sort of visual indicator but we already have a tooltip around the labels/values so it might be a bit too much. I've mentioned this to @lukasztyrala. 

I have a check to show the pointer cursor only if the clipboard is available. I haven't updated the check now that I added the fallback method, but the api seems to be pretty widely supported.

Reply:
Hey Adela, thanks for adding a fallback. It's a site by site setting in Chrome, quick video below showing disabling it 😄 

https://github.com/grafana/grafana/assets/199847/b6458624-c613-4d7a-919d-a6314ace75dc

I tested it out and the fallback appears to work as well 🥳 . Great stuff!

Reply:
Ohh, thanks! Google is behind on stuff.. 😅

Reply:
Yo, I learn something new every day!